src/ethereum/arrow_glacier/spec.py

```diff
@@ -571,12 +571,12 @@ def apply_body(
 
     pay_rewards(state, block_number, coinbase, ommers)
 
-    gas_remaining = block_gas_limit - gas_available
+    gas_used = block_gas_limit - gas_available
```

Comment:
There are some [mypy errors](https://github.com/ethereum/execution-specs/actions/runs/4216276148/jobs/7343499883#step:7:25). I believe the fix is to:

```suggestion
    block_gas_used = block_gas_limit - gas_available
```

Since `gas_used` was defined earlier, with a different type.